src/wrappers/express.js

```diff
@@ -37,9 +37,6 @@ function expressMiddleware(req, res, next) {
         // Handle response
         const requestPromise = new Promise((resolve) => {
             res.once('finish', function handleResponse() {
-                if (!req.route) {
```

Comment:
I wouldn't remove that, because that reduces lots of non-existing endpoints (404) which result in a spammy traces.
I would put it as an ENV instead

Reply:
Agree. fixed